docs/source/release_notes.rst

```diff
@@ -41,7 +49,7 @@ Release Notes
 .. warning::
 
     **Breaking Changes**
-        * DefaultDataChecks now accepts a problem_type parameter that must be specified :pr:`1167`
+        * `DefaultDataChecks` now accepts a `problem_type` parameter that must be specified :pr:`1167`
         * Pipeline's `._transform` method to evaluate all the preprocessing steps of a pipeline has been replaced with `.compute_estimator_features` :pr:`1231`
```

Comment:
This isn't technically breaking since its a private API. But, its ok!

Reply:
Ahaha yeah, @freddyaboulton and I thought since this could have been used that it'd be good to make note of it 👀 I can remove it though!

Reply:
Yeah that's a good point, keep it!